realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -781,6 +781,7 @@ public void realmConfiguration_fileName() {
     }
 
     @Test
+    @Ignore
```

Comment:
actually based on what the test want to check, we might be able to make it much faster, eg.: do not do queries, just write to the object then read and check.